core/state/state_object.go

```diff
@@ -160,16 +160,20 @@ func (c *stateObject) getTrie(db Database) Trie {
 }
 
 // GetState returns a value in account storage.
```

Comment:
Comment should say that it returns now dirty flag, too

Reply:
Updated.